### PR TITLE
Fixed an issue doesn't show save map title in change popover modal

### LINKED ISSUE #358: [MetaMaps] Doesn't save map name
Currently, when change map name in the map view, it is updating the map name on the Title. however, it doesn't save this map name in the `graphql`

### PR DESCRIPTION
**Pull request by ace-smart**
Fixes  #358
Fixed an issue doesn't show save map title in change popover modal.
And then, updated to show the default existing map title when open the popup modal

### CODE REVIEW

File: packages/metamaps/pages/[author_address]/map/[id].tsx [hunk 1]
```diff
@@ -78,6 +78,8 @@ export const MapPageComponent: FC<MapPageProps> = ({ dispatch }) => {
     data: items,
   });
 
+  dispatch({ type: 'POPOVER_NAME_INPUT', value: data?.Map_by_pk.name });
```
Comment (pakokrew) [inline packages/metamaps/pages/[author_address]/map/[id].tsx:81]: People working on this project should learn about render mechanisms of React and how works hooks.

The content of this method, from line 18 to line 88, is executed at each render. 
It means that:
- the dispatches will be sent many times
- the items will be computed many times

You must wrap this logic into respectively useEffect and useMemo

**Review by pakokrew — APPROVED** [review]

**Review by alalonde — CHANGES_REQUESTED** [review]
Please change to leverage `useEffect` and `useMemo` appropriately, and rebase from master.

**Comment by dan13ram** [comment]
No longer necessary as we have deprecated `metamaps`

**Comment by dan13ram** [comment]
Closing this since we are no longer working on `metamaps`. Please reopen if needed in the future.

**Pull request closed without merging**